snow/engine/snowman/block/block_context_vm.canoto.go

```diff
@@ -1,12 +1,14 @@
 // Code generated by canoto. DO NOT EDIT.
 // versions:
-// 	canoto v0.10.0
-// source: block_context_vm.go
+// 	canoto v0.13.3
+// source: ./snow/engine/snowman/block/block_context_vm.go
```

Comment:
Seems like this was generated differently than it was before (with `go generate`)